prime-router/src/main/resources/db/migration/V64__update_list_send_failure.sql

```diff
@@ -0,0 +1,88 @@
+/*
+* The Flyway tool applies this migration to create the database.
+*
+* Follow this style guide https://about.gitlab.com/handbook/business-ops/data-team/platform/sql-style-guide/
+* use VARCHAR(63) for names in organization and schema
+*
+* Copy a version of this comment into the next migration
+*
+*/
+
+/*
+    This is the "Last Mile" failures query.
+    See https://github.com/CDCgov/prime-reportstream/issues/9649
+ */
+DROP FUNCTION IF EXISTS list_send_failures;
+CREATE OR REPLACE FUNCTION list_send_failures(days_to_show_param INT)
+    RETURNS TABLE
+            (
+                action_id            BIGINT,
+                report_id            UUID,
+                receiver             TEXT,
+                file_name            TEXT,
+                failed_at           TIMESTAMP WITH TIME ZONE,
+                action_params        varchar(2048),
+                action_result        TEXT,
+                body_url             varchar(2048),
+                report_file_receiver TEXT
+            )
+    LANGUAGE plpgsql
+AS
+$$
+
+BEGIN
+    RETURN QUERY
+        WITH send_failures as
+                 (select a.action_id
+                       , split_part(a.action_params, '&', 3)::uuid
+                         as "report_id"
+                       , a.created_at as "failed_at"
+                       , a.action_params
+                       , a.action_result
+                       , case
+                             when strpos(a.action_result, 'unrecoverable exception') > 0 and
+                                  strpos(a.action_result, 'host=') > 0
+                                 then trim(split_part(
+                                     split_part(a.action_result, 'host=', 2), ',', 1))
+                             when strpos(a.action_result, 'unrecoverable exception') > 0 and
+                                  strpos(a.action_result, 'database:') > 0
+                                 then trim(split_part(
+                                     split_part(a.action_result, 'database:', 2), 'Event:', 1))
+                             when strpos(a.action_result, 'AS2') > 0
+                                 then trim(split_part(
+                                     split_part(a.action_result, 'orgService =', 2),
+                                     ');', 1))
+                            when strpos(a.action_result, 'orgService = ') > 0
+                                 then trim(split_part(
+                                     split_part(a.action_result, 'orgService =', 2),
+                                     '),', 1))
+                             when strpos(a.action_result, 'gaen') > 0
+                                 then 'wa-phd.gaen'
```

Comment:
This also seems weirdly specific, and feels like maybe it should be a change to the way we're logging. Anything that says gaen will automatically be attributed to Washington? I know nobody else uses it right now, but feels weird to hard code it like this.